tools/cloudera/build_hue_common.sh

```diff
@@ -228,8 +239,9 @@ function centos7_install() {
     sudo -- sh -c 'curl -sSLO https://dev.mysql.com/get/mysql80-community-release-el7-11.noarch.rpm && \
         rpm -ivh mysql80-community-release-el7-11.noarch.rpm && \
         yum install -y mysql-community-libs mysql-community-client-plugins mysql-community-common'
-    # NODEJS 14 install
-    sudo -- sh -c 'yum install -y rh-nodejs14-nodejs'
+    # NODEJS 16 install
```

Comment:
Update comment to say why we use v 16

Reply:
Added a comment to explain why we are moving to v16.